cli/src/cmd/init.js

```diff
@@ -33,22 +33,80 @@ export async function init() {
     });
 
     if (await folderExists(projectName)) {
-      spinner.fail(
-        `Target directory "${targetDir}" already exists. Remove it or choose a different name.`
+      throw Error(
+        `Target directory "${projectName}" already exists. Remove it or choose a different name.`
       );
-      process.exit(1);
     }
 
+    const INIT_BASIC = 'basic';
+    const INIT_ADVANCED = 'advanced';
+    const { initType } = await spinner.prompt({
+      type: 'select',
+      name: 'initType',
+      message: 'What kind of project do you want to init?',
+      choices: [
+        {
+          title: 'hello-world',
```

Comment:
what do you think about using PascalCase here ? 
```suggestion
          title: 'HelloWorld',
```

Reply:
plain English seems even better :)

```suggestion
          title: 'Hello World',
```